update zedonk.md (#357)
cyclr/docs: pages/connector-authentication/zedonk.md

## Cyclr setup

### Account setup

> When Installing the Zedonk connector within an account, you will be asked for the following credentials:

Reviewer: ```suggestion
When you install the Zedonk connector within an account, you need to enter the following credentials:
```
tense and capitalisation
